src/lib/config/lib-config.model.ts

```diff
@@ -1,9 +1,34 @@
 'use strict';
 
+export const CONTENT_DEFAULT = '';
+export const RECURSIVE_DEFAULT = true;
+export const SORT_MODULES_DEFAULT = true;
+export const DRY_RUN_DEFAULT = false;
+export const DIRECTORY_PATH_DEFAULT = '.';
+export const FILES_DEFAULT: string[] = [];
+export const PRINT_OUTPUT_DEFAULT = false;
+export const VERBOSE_DEFAULT = false;
+
+/**
+ * The default sorting config
+ */
+export class DefaultSortingConfig implements SortingConfig {
+  content = CONTENT_DEFAULT;
+  directoryPath = DIRECTORY_PATH_DEFAULT;
+  recursive = RECURSIVE_DEFAULT;
+  sortModules = SORT_MODULES_DEFAULT;
+  files = FILES_DEFAULT;
+  dryRun = DRY_RUN_DEFAULT;
+  printOutput = PRINT_OUTPUT_DEFAULT;
+  verbose = VERBOSE_DEFAULT;
+}
+
 /**
  * A config for sorting
  */
 export interface SortingConfig {
+  /** a file's content */
+  content: string;
```

Comment:
Why is this one mandatory?

Reply:
Our CLI supports imports to sort inline, and for that we use `content`.
We also reuse it when iterating through `filePaths` [here](https://github.com/znikola/es6-typescript-import-sorter/pull/13/files#diff-c9ee77b8b3575c61b785ba838750243bR36)

Do you think we can do this in a better way?